ext/sdk/resources/sdk-root/shell/src/resource-templates/ts/scaffolder.ts

```diff
@@ -0,0 +1,230 @@
+import { inject, injectable } from "inversify";
+import { FsService } from "backend/fs/fs-service";
+import * as cp from 'child_process';
+import { concurrently } from "utils/concurrently";
+import { ResourceTemplateScaffolder, ResourceTemplateScaffolderArgs } from "../types";
+
+@injectable()
+export default class TsScaffolder implements ResourceTemplateScaffolder {
+  @inject(FsService)
+  protected readonly fsService: FsService;
+
+  async scaffold({ request, manifest, resourcePath }: ResourceTemplateScaffolderArgs) {
+    const resourceName = request.assetName;
+    
+    manifest.clientScripts.push('dist/client/client.js');
+    manifest.serverScripts.push('dist/server/*.server.js');
+
+    manifest.fxdkWatchCommands.push(['yarn', [
+      'webpack', '--mode', 'development', '--watch'
+    ]]);
+    manifest.fxdkWatchCommands.push(['yarn', [
```

Comment:
Should this not be `fxdkBuildCommands`?

Reply:
As far as I am concerned, that is never used, in any of the scaffolders. Although I am not sure, other than that the PR is approved, so far. I guess @nihonium-cfx would have to give an answer on that 😄

Reply:
Good catch, it should. Same as there should be build commands in cs template, oops.

Reply:
Great! I'll fix that :D

Reply:
Also, if `fxdkWatchCommands` and `fxdkBuildCommands` are going to work in the way I suspect they're going to work, you might eventually want to add a `touch .yarn.installed` command into the watch/build to stop FiveM trying to run `yarn install`.